releasenotes/notes/profiling-fix-profiler-copy-6d775a134ec2233f.yaml

```diff
@@ -0,0 +1,9 @@
+---
+upgrade:
+  - |
+    The profiler ``asyncio_loop_policy`` attribute has been renamed to
```

Comment:
This is an internal attribute, right? Do we need to document the change at all?

Reply:
It's in the public API for the `Profiler` object, so in the unlikely case anyone used it, it's better to announce it.

Reply:
Ah I see, it's on `_ProfilerInstance` but all `_ProfilerInstance`'s attributes are exposed through `Profiler` due to `Profiler.__getattr__`. 👍 

Still feels like we're making a breaking change but we can probably hand-wave it here: https://ddtrace.readthedocs.io/en/stable/versioning.html#interfaces since it's a dynamic lookup on the class.